### PR TITLE
Add application field to the PerfCompare endpoint response 

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**
In order to display the application in PerfCompare we need to update the endpoint to provide this information for base and new values.

For testing:
[Staging](https://treeherder.allizom.org/perfherder/compare?originalProject=mozilla-central&originalRevision=059b77df2394c7caf1061e1b817e004878738efa&newProject=mozilla-central&newRevision=e5ba3b52ebac2d2582cecd6122443fc024cd24c9&framework=1&page=11) vs [Local response from the PerfCompare endpoint - connected to the staging db](http://localhost:8000/api/perfcompare/results/?base_repository=mozilla-central&new_repository=mozilla-central&framework=1&interval=604800&no_subtests=true&base_revision=059b77df2394c7caf1061e1b817e004878738efa&new_revision=e5ba3b52ebac2d2582cecd6122443fc024cd24c9)

### CODE REVIEW

File: treeherder/webapp/api/performance_serializers.py [hunk 1]
```diff
@@ -443,6 +443,14 @@ def validate(self, data):
 class PerfCompareResultsSerializer(serializers.ModelSerializer):
     base_rev = serializers.CharField()
     new_rev = serializers.CharField()
+    base_app = serializers.CharField(
+        max_length=10,
```
Comment (alexandru-io) [inline treeherder/webapp/api/performance_serializers.py:447]:
```suggestion
        max_length=25,
```

File: treeherder/webapp/api/performance_serializers.py [hunk 2]
```diff
@@ -443,6 +443,14 @@ def validate(self, data):
...
+        default='',
+    )
+    new_app = serializers.CharField(
+        max_length=10,
```
Comment (alexandru-io) [inline treeherder/webapp/api/performance_serializers.py:451]: ditto
```suggestion
        max_length=25,
```

**Comment by alexandru-io** [comment]
> In order to display the application in PerfCompare we need to update the endpoint to provide this information for base and new values.
> 
> For testing: [Staging](https://treeherder.allizom.org/perfherder/compare?originalProject=mozilla-central&originalRevision=059b77df2394c7caf1061e1b817e004878738efa&newProject=mozilla-central&newRevision=e5ba3b52ebac2d2582cecd6122443fc024cd24c9&framework=1&page=11) vs [Local response from the PerfCompare endpoint - connected to the staging db](http://localhost:8000/api/perfcompare/results/?base_repository=mozilla-central&new_repository=mozilla-central&framework=1&interval=604800&no_subtests=true&base_revision=059b77df2394c7caf1061e1b817e004878738efa&new_revision=e5ba3b52ebac2d2582cecd6122443fc024cd24c9)

**Review by alexandru-io — APPROVED** [review]
(I'm sorry, I must have pressed something and closed your PR by mistake.)

Here, the results from my localhhost:
image

**Comment by beatrice-acasandrei** [comment]
> > In order to display the application in PerfCompare we need to update the endpoint to provide this information for base and new values.
> > For testing: [Staging](https://treeherder.allizom.org/perfherder/compare?originalProject=mozilla-central&originalRevision=059b77df2394c7caf1061e1b817e004878738efa&newProject=mozilla-central&newRevision=e5ba3b52ebac2d2582cecd6122443fc024cd24c9&framework=1&page=11) vs [Local response from the PerfCompare endpoint - connected to the staging db](http://localhost:8000/api/perfcompare/results/?base_repository=mozilla-central&new_repository=mozilla-central&framework=1&interval=604800&no_subtests=true&base_revision=059b77df2394c7caf1061e1b817e004878738efa&new_revision=e5ba3b52ebac2d2582cecd6122443fc024cd24c9)
> 
> You're providing a link to compare view (which you haven't touched in the PR as far as I can tell) and an API response from PerCompare from localhost. I am pulling the 2 compared commits from mozilla-central to see the response locally. Is the compare view link relevant?

The response from the PerfCompare endpoint should contain the data that is being displayed on Perfherder's Compare View.

**Comment by beatrice-acasandrei** [comment]
@Archaeopteryx I would like to merge this. Do you know when that will be possible since `ci/circleci: python-tests-postgres` is currently failing?

**Comment by Archaeopteryx** [comment]
The failure should be resolved if the patch gets rebased on latest master.

**Pull request merged**